resolver/resolver.cc

```diff
@@ -1144,25 +1144,46 @@ class ResolveConstantsWalk {
         ENFORCE(block->body);
 
         auto blockLoc = core::Loc(todo.file, block->body.loc());
-        auto *id = ast::cast_tree<ast::ConstantLit>(block->body);
+        core::ClassOrModuleRef symbol = core::Symbols::StubModule();
 
-        if (id == nullptr || !id->symbol.exists() || !id->symbol.isClassOrModule()) {
+        if(auto *constant = ast::cast_tree<ast::ConstantLit>(block->body)) {
+            if(constant != nullptr && constant->symbol.exists() && constant->symbol.isClassOrModule()) {
```

Comment:
```suggestion
            if (constant->symbol.exists() && constant->symbol.isClassOrModule()) {
```

(The previous `cast_tree` will have ensured that we have a non-null pointer here.)